testing/e2e.sh

```diff
@@ -64,6 +64,11 @@ docker_build(
   base = "@pause//image",
   workdir = "/tmp",
 )
+
+docker_build(
```

Comment:
So this file is generally for testing stuff we can't easily test within Bazel, e.g. the behavior of `bazel run :foo` etc.  You should be able to `container_load(name="...", file="//testdata:pause.tar")` from our `WORKSPACE`, do something with it in `testdata/BUILD`, incorporate the new image into `container/BUILD`s `TEST_DATA` and then inspect the result in `container/image_test.py`.

Sorry for the confusion around the different kinds of testing :(

Reply:
Done. Thanks for the instructions.